dev/benchmarks/complex_layout/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Comment:
I notice this changes fron ***all.zip to ***bin.zip. Is there one we should generally prefer, or does it depend on the situation? 

As I understand, its the difference of whether sources/documentation come bundled, but I'm not sure what the implications of that for all these gradle wrappers in /benchmarks, for example.

Reply:
I think for normal users all is probably safe since the size is not large. That said this is what is in the 3.10 branch and we dont want to make any changes.